[2025-09-11T06:13:46.157+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: aviationstack_kafka_dag.run_bronze_load manual__2025-09-11T06:13:44.164179+00:00 [queued]>
[2025-09-11T06:13:46.159+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: aviationstack_kafka_dag.run_bronze_load manual__2025-09-11T06:13:44.164179+00:00 [queued]>
[2025-09-11T06:13:46.159+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-09-11T06:13:46.163+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_bronze_load> on 2025-09-11 06:13:44.164179+00:00
[2025-09-11T06:13:46.165+0000] {standard_task_runner.py:60} INFO - Started process 187 to run task
[2025-09-11T06:13:46.167+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'aviationstack_kafka_dag', 'run_bronze_load', 'manual__2025-09-11T06:13:44.164179+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/aviationstack_dag.py', '--cfg-path', '/tmp/tmpjcub_01j']
[2025-09-11T06:13:46.168+0000] {standard_task_runner.py:88} INFO - Job 4: Subtask run_bronze_load
[2025-09-11T06:13:46.188+0000] {task_command.py:423} INFO - Running <TaskInstance: aviationstack_kafka_dag.run_bronze_load manual__2025-09-11T06:13:44.164179+00:00 [running]> on host d46a8dc9e93f
[2025-09-11T06:13:46.216+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='aviationstack_kafka_dag' AIRFLOW_CTX_TASK_ID='run_bronze_load' AIRFLOW_CTX_EXECUTION_DATE='2025-09-11T06:13:44.164179+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-11T06:13:44.164179+00:00'
[2025-09-11T06:13:46.228+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2025-09-11T06:13:46.230+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 199, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 216, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/etl/bronze/avstack/bronze_load.py", line 29, in insert_into_bronze_ddl
    conn = hook.get_conn()
           ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/postgres/hooks/postgres.py", line 158, in get_conn
    self.conn = psycopg2.connect(**conn_args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.OperationalError: connection to server at "postgres" (172.18.0.3), port 5432 failed: FATAL:  ***word authentication failed for user "postgres"

[2025-09-11T06:13:46.234+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=aviationstack_kafka_dag, task_id=run_bronze_load, execution_date=20250911T061344, start_date=20250911T061346, end_date=20250911T061346
[2025-09-11T06:13:46.238+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 4 for task run_bronze_load (connection to server at "postgres" (172.18.0.3), port 5432 failed: FATAL:  ***word authentication failed for user "postgres"
; 187)
[2025-09-11T06:13:46.263+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-09-11T06:13:46.274+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
